### DISCUSSION TITLE
Is there a way to switch camera front to back or back to front when using vision ?

### DISCUSSION THREAD

**Discussion by phicha20224** [open-webui/open-webui#4967] [Ideas]
Is there a way to switch camera front to back or back to front when using vision ?
I know open-webui its great already. just maybe a idea. Thank You.

**Comment by justinh-rahb** [comment]
This:
<img width="295" alt="Screenshot 2024-08-29 at 8 19 20 AM" src="https://github.com/user-attachments/assets/71cde906-f7d8-4f3d-97e6-4c9ab1de6d29">

**Reply by phicha20224** [reply]
it is workingg Thank You very much.
